examples/features/multiplex/README.md

```diff
@@ -0,0 +1,11 @@
+# Multiplex
+
+A `ClientConn` can be shared by two stubs and two servicers can share a server.
```

Comment:
Add something like: "This example illustrates how to perform both types of sharing."

Reply:
Done